specification/vi/resource-manager/Microsoft.VideoIndexer/preview/2021-04-01-preview/vi.json

```diff
@@ -0,0 +1,844 @@
+          "$ref": "#/definitions/MediaServices"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Gets the status of the account at the time the operation was called.",
+          "enum": [
+            "Succeeded",
+            "Failed",
+            "Canceled",
+            "Accepted",
+            "Provisioning",
+            "Deleting"
+          ],
+          "x-ms-enum": {
+            "name": "Succeeded",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "AccountList": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Account"
+          },
+          "description": "List of accounts and their properties."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "URL to get the next set of operation list results if there are any.",
+          "example": "https://management.azure.com/{accountsOperation}?api-version={version}&%24skiptoken={token}"
+        }
+      },
+      "description": "The list operation response, that contains the data pools and their properties."
+    },
+    "GenerateAccessTokenParameters": {
+      "description": "Access token generation request's parameters",
+      "required": [
+        "permissionType",
+        "scope"
+      ],
+      "properties": {
+        "permissionType": {
+          "readOnly": false,
+          "type": "string",
+          "description": "The requested permission",
+          "enum": [
+            "Contributor",
+            "Reader"
+          ],
+          "x-ms-enum": {
+            "name": "Contributor",
+            "modelAsString": true
+          }
+        },
+        "scope": {
+          "readOnly": false,
+          "type": "string",
+          "description": "The requested media type",
+          "enum": [
+            "Video",
+            "Account",
+            "Project"
+          ],
+          "x-ms-enum": {
+            "name": "Video",
+            "modelAsString": true
+          }
+        },
+        "videoId": {
+          "description": "The video ID",
+          "type": "string",
+          "readOnly": false,
+          "example": "07ec9e38d4"
+        },
+        "projectId": {
+          "description": "The project ID",
+          "type": "string",
+          "readOnly": false,
+          "example": ""
+        }
+      }
+    },
+    "MediaServices": {
+      "description": "The media services details",
+      "type": "object",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "The media services resource id",
+          "readOnly": false,
+          "example": "/subscriptions/xxx/resourceGroups/contoso-videoanalyzer-rg/providers/Microsoft.Media/mediaservices/contoso-videoanalyzer-ms"
+        },
+        "userAssignedManagedIdentity": {
```

Comment:
This is not the pattern used in ARM for specifying MSIs. There should be a top-level property by name 'identity'.
For example refer spec https://github.com/Azure/azure-rest-api-specs/blob/1b0ed8edd58bb7c9ade9a27430759527bd4eec8e/specification/resources/resource-manager/Microsoft.Resources/stable/2020-10-01/deploymentScripts.json#L420

Reply:
fixed